install/helm/open-match/templates/om-configmap-default.yaml

```diff
@@ -58,7 +58,7 @@ data:
         grpcport: "{{ .Values.frontend.grpcPort }}"
         httpport: "{{ .Values.frontend.httpPort }}"
       query:
-        hostname: "{{ .Values.query.hostName }}.{{ .Release.Namespace }}.svc.cluster.local"
+        hostname: "{{ .Values.query.hostName }}"
```

Comment:
The demo doesn't use the config maps: https://github.com/googleforgames/open-match/blob/master/examples/functions/golang/soloduel/main.go
Nothing is going to read this across namespace boundaries, because k8s doesn't support binding them as such.

Tested demo anyways and it still works.